### PR TITLE
Remove unused params in ServeHTTP and handleHelp

### PR DESCRIPTION
**Pull request by mickmister**

### CODE REVIEW

File: server/api.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func checkPluginRequest(next http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
-func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
+func (p *Plugin) ServeHTTP(_ *plugin.Context, w http.ResponseWriter, r *http.Request) {
```
Comment (hanzei) [inline server/api.go:158]: Do we want to tune the linter to not warn about this stuff? It doesn't seem helpful to me.

**Review by hanzei — APPROVED** [review]

File: server/api.go (hunk 1 above)
Comment (mickmister) [reply server/api.go:158]: @hanzei Sure, this has caused the same problem in every other plugin. Can this be a centralized change, or does this need to be applied to each repo individually?

Comment (hanzei) [reply server/api.go:158]: I don't think we can fix this per repo.

Comment (mickmister) [reply server/api.go:158]: @hanzei Would you happen to know which repo the fix goes into? Possibly `actions`?

Comment (hanzei) [reply server/api.go:158]: Sorry, my previous comment was ambiguous. I meant that we must update `.golangci.yml` for every repository.

**Pull request merged**